examples/design-system/src/components/Separator/Separator.tsx

```diff
@@ -0,0 +1,12 @@
+import { Separator } from '@jeromefitz/design-system/src/components'
+
+const _Separator = () => {
+  return (
+    <Separator
+      aria-hidden={true}
+      css={{ my: '1rem !important', width: '100% !important' }}
+    />
+  )
+}
```

Comment:
Please remove and add a new `margin` variant, and additional `size` variant to `Separator`.

Instead of `aria-hidden={true}` by passing `decorative` prop to `Separator` that will have the intention desired.